src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -13,34 +15,40 @@ public class FlywheelSubsystem extends SubsystemBase {
      * 1 Follower TalonFX (Right)
      * Total: 2 TalonFX motors
      */
-
+    TalonFX adi;
```

Comment:
Change this to private TalonFX <variable name>, because this variable should only be changed from the methods set in the subsystem class